pinot-plugins/pinot-stream-ingestion/pinot-kinesis/pom.xml

```diff
@@ -36,7 +36,7 @@
   <properties>
     <pinot.root>${basedir}/../../..</pinot.root>
     <phase.prop>package</phase.prop>
-    <aws.version>2.14.28</aws.version>
+    <aws.version>2.17.207</aws.version>
```

Comment:
We can make aws.version as well common root pom.xml. We will need to keep 2 versions though. 
`aws.version` and `aws.version.v2`